index.html

```diff
@@ -850,19 +851,14 @@
             </li>
             <li>Let <em>P</em> be a new <a>Promise</a>.
             </li>
-            <li>Return <em>P</em>.
+            <li>Return <em>P</em>, but continue running these steps in
+            parallel.
```

Comment:
I find the term "in parallel" confusing personally, but I used it as defined in HTML 5.1 to mean "execute the subsequent steps in order, although at the same time as other logic in the specification":
http://www.w3.org/TR/html51/infrastructure.html#in-parallel

I think "in parallel" should link back to its definition, as other terms in the spec. I'll update the PR accordingly.